Assets/Scenes/SampleScene.unity

```diff
@@ -1,267 +0,0 @@
-%YAML 1.1
-%TAG !u! tag:unity3d.com,2011:
---- !u!29 &1
-OcclusionCullingSettings:

```

Comment:
Why is this also deleted in this branch? This was deleted in the guard setup branch too. We should not have double changes.